src/main/kotlin/graphics/scenery/geometry/curve/PartialCurve.kt

```diff
@@ -0,0 +1,28 @@
+package graphics.scenery.geometry.curve
+
+import graphics.scenery.BufferUtils
+import graphics.scenery.Mesh
+import java.nio.FloatBuffer
+
+/**
+ * Each child of the curve must be, per definition, another Mesh. Therefore, this class turns a List of
+ * vertices into a Mesh.
+ *
+ * @param verticesBuffer: corresponds to the vertices buffer of Mesh
+ * @param normalVectors: corresponds to the normals buffer of Mesh
+ *
+ */
+class PartialCurve(verticesBuffer: FloatBuffer, normalVectors: FloatBuffer) : Mesh("PartialCurve") {
```

Comment:
PartialCurve should maybe be renamed PartialCurveMesh? I feel the current name kinda gives the (wrong) impression it's a kind of `Curve`.